equinox/nn/_embedding.py

```diff
@@ -354,8 +352,7 @@ def get_positional_encoding(
         pos = jnp.arange(seq_len, dtype=float)[:, jnp.newaxis]
```

Comment:
Heads-up that you still seem to have some `dtype=float`s

Reply:
Oops, must have overlooked this, my bad. It's fixed now